app/src/test/java/com/tvtracker/IntegrationTests.kt

```diff
@@ -38,8 +36,8 @@ class IntegrationTests {
         Assert.assertNotNull(imdbResponse!!.results)
 
         var foundBattleBots = false
-        imdbResponse!!.results!!.forEach {
-            if(it.title.uppercase().equals("BATTLEBOTS")){
+        imdbResponse!!.results.forEach {
+            if(it.title.uppercase()==("BATTLEBOTS")){
```

Comment:
Using == is better used in Kotlin for Structural Equality and is suggested for best practices. 
https://kotlinlang.org/docs/equality.html